.github/workflows/package-binaries.yml

```diff
@@ -62,22 +68,34 @@ jobs:
     - name: get ctrDetect
       run: |
         echo "test test trying to build for ${{ matrix.os }}"
+        mkdir -p pkg
         case "${{ matrix.os }}" in
           linux)
             URL="http://www.creatis.insa-lyon.fr/~sdika/soft/ctrDetect-v1_x86_64.tar.gz"
+            curl -L "$URL" -o ctrDetect.tar.gz
+            tar -zxvf ctrDetect.tar.gz
+            cp -p ctrDetect/{spine_detect,spine_train_svm} pkg/
+            mv pkg/spine_train_svm pkg/train_svm  # we have this one named unusually
             ;;
           osx)
             URL="http://www.creatis.insa-lyon.fr/~sdika/soft/ctrDetect-v1_macos10.11.tar.gz"
+            curl -L "$URL" -o ctrDetect.tar.gz
+            tar -zxvf ctrDetect.tar.gz
+            cp -p ctrDetect/{spine_detect,spine_train_svm} pkg/
+            mv pkg/spine_train_svm pkg/train_svm  # we have this one named unusually
```

Comment:
I see, these lines are a repeat of the `linux)` case, but they can't be factored out because the `windows)` case is too different. Makes sense.